app/helpers/accessible_format_helper.rb

```diff
@@ -0,0 +1,24 @@
+module AccessibleFormatHelper
+  def accessible_format_options_to_radio_items(checked = nil)
+    values = I18n.translate("models.accessible_format_options").map do |item|
+      radio_item = item.dup
```

Comment:
It definitely fails without dup-ing - I think modifying the input parameter into a map and returning it as the mapped value is problematic? But I admit this is ruby esoteria that I'm not 100% sure on.

Reply:
Hmm, I've tested this out in the console and can't see any failures - could you share the error you're seeing?

<img width="1270" alt="Screenshot 2022-03-09 at 11 01 13" src="https://user-images.githubusercontent.com/24479188/157428928-0a44836d-7d76-4d8f-b8dc-e8e7ad1a1926.png">

Reply:
Well, good question - what I'm seeing is the test suite failing. I suspect the reason it's working for you is that you're not calling render_conditional. So it's something there (perhaps the in-code render does something unsafe with the variables?)

Honestly, I suspect the smart thing to do would be to get rid of all this and just have the options straight in the view. I feel like what I've done here is something "clever", and it would be better to just do it in an idiomatic way.

Reply:
Conclusion to the conversation on Slack is that we're leaving the implementation of this method as is for now but looking to simplifying it next week (feature is going out tomorrow).